torch/_functorch/partitioners.py

```diff
@@ -389,6 +393,18 @@ def is_tensor_node(x):
 
     AGGRESSIVE_RECOMPUTATION = False
 
+    def is_materialized_backwards(node):
```

Comment:
nit: it took me a while to understand that this means: only computed during backward and not during forward. A small comment would be nice.